ruby-challenges.rb

```diff
@@ -19,17 +25,62 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
-
+def sum (nums_array)
+  nums_array.inject(:+)
+end
+p sum(nums_array1)
+p sum(nums_array2)
```

Comment:
Inject is a cool and unique approach.  .sum is more commonly used for this type of problem as it is less processing time however it is cool to know and explore the different options Ruby has to offer.